midikeyboard/src/main/AndroidManifest.xml

```diff
@@ -1,12 +1,9 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="com.mobileer.midikeyboard"
-        android:versionCode="4"
-        android:versionName="@string/versionName">
```

Comment:
We lost a feature. The versionName was grabbed from strings.xml. But I suppose gradle may not support this.